chef-dk.gemspec

```diff
@@ -30,7 +30,9 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.0'
 
-  gem.files = %w(Gemfile Rakefile LICENSE README.md CONTRIBUTING.md warning.txt) + Dir.glob("*.gemspec") +
+  gem.files = %w(Rakefile LICENSE README.md CONTRIBUTING.md warning.txt) +
```

Comment:
A little confused why we omit the Gemfile from the package but keep it in source